css/styles.css

```diff
@@ -0,0 +1,399 @@
+    flex-grow: 0;
+}
+
+.list-item-abts>h3 {
+    font-size: 24px;
+    line-height: 30px;
+    text-transform: capitalize;
+}
+
+.list-item-abts>p {
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.readmore-about-us {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+    color: #fff;
+}
+
+.how-to-work-hdr {
+    font-size: 44px;
+    line-height: 56px;
+    text-align: center;
+    text-transform: capitalize;
+}
+
+.step-how-to {
+    list-style: none;
+}
+
+.step-how-to, dt {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 136%;
+    text-transform: capitalize;
+    text-align: center;
+}
+
+.step-how-to, dd {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 133%;
+    text-transform: capitalize;
+    text-align: center;
+    color: #fff;
+}
+
+.r-map-hdr {
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+}
+
+.phase-step {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 36px;
+    line-height: 122%;
+    text-transform: capitalize;
+    color: #fff;
+}
+
+.phase-step-item {
+    font-size: 18px;
+    line-height: 156%;
+    color: #c2c3c5;
+}
+
+
+/*____________________?____________________________*/
+
+/*
+.r-map-list-icons~.button {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 140%;
+    text-transform: capitalize;
+    color: #fff;
+}
+*/
+
+/*____________________?____________________________*/
+
+.our-team-hdr {
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+}
+
+.card-name-prof>dt {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 125%;
+    text-transform: capitalize;
+    text-align: center;
+    color: #fff;
+}
+.card-name-prof>dd {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 156%;
+    text-align: center;
+    color: #c2c3c5;
+}
+
+.partner-hdr {
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+}
+
+.faq-hdr {
+font-size: 44px;
+line-height: 127%;
+text-transform: capitalize;
+text-align: center;
+}
+
+.faq-sum {
+    font-family: "Bakbak One", sans-serif;
+font-weight: 400;
+font-size: 24px;
+line-height: 125%;
+text-transform: capitalize;
+color: #fff;
+}
+
+.details-faq>p {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 156%;
+    color: #c2c3c5;
+}
+
+.subscribe-hdr>h2 {
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+}
+
+.subscribe-hdr>p {
+    font-family: "Inter", sans-serif;
```

Comment:
It looks like a design bug, let's apply the Poppins font here, which is used for the rest of the plain text?